### ISSUE TITLE
Row selection callback

### ISSUE THREAD

**Issue by vladjerca** [iamisti/mdDataTable#169]
This is a dupe of #156 

As I mentioned in the previous ticket, even on the codepen provided: http://codepen.io/iamisti/full/OMvbMj/

You can clearly see that when the header checkbox is ticked the callback is not triggered.

The header does say '10 items selected', but you have no way of accessing (that I know of) the selected items if you select/deselect all.

I don't want to be insistent, but I think the previous ticket was closed without an actual solution.

__Update__: To be more precies:

``` js
                $scope.selectAllRows = false;

                $scope.$watch('selectAllRows', function(val){
                    ctrl.tableDataStorageService.setAllRowsSelected(val, $scope.isPaginationEnabled());
                    // selectedRowCallback needs to be triggered here as well
                });
```

**Comment by iamisti** [comment]
@vladjerca hi man, sorry if I closed your ticket without providing a solution for you.
I'll have a look at this ticket today or tomorrow and will give you an answer.

**Comment by iamisti** [comment]
Fixed with v1.6.12
Please have a look at it and comment here if it doesn't work well for your needs.

**Comment by vladjerca** [comment]
thank you!
